compiler/rustc_mir_build/src/thir/print.rs

```diff
@@ -519,6 +519,19 @@ impl<'a, 'tcx> ThirPrinter<'a, 'tcx> {
                 self.print_inline_asm_expr(&**expr, depth_lvl + 2);
                 print_indented!(self, "}", depth_lvl);
             }
+            OffsetOf { container, fields } => {
+                print_indented!(self, "InlineAsm {", depth_lvl);
+                print_indented!(self, format!("container: {:?}", container), depth_lvl + 1);
+                print_indented!(self, "fields: [", depth_lvl + 1);
+
+                for field in fields.iter() {
+                    print_indented!(self, format!("{:?}", field), depth_lvl + 2);
+                    print_indented!(self, ",", depth_lvl + 1);
```

Comment:
I'm not entirely sure how all of this works, but shouldn't this be
```suggestion
                    print_indented!(self, format!("{:?},", field), depth_lvl + 2);
```

Reply:
Everything else seems to have the commas be at a lower depth level than what's around them

Reply:
Okay, interesting